src/containerapp/azext_containerapp/_cloud_build_utils.py

```diff
@@ -130,27 +130,32 @@ def spin():
 
         # Source code compression
         done_spinner = False
-        thread = display_spinner(f"Compressing data: {font_bold}{source}{font_default}")
-        tar_file_path = os.path.join(tempfile.gettempdir(), f"{build_name}.tar.gz")
-        archive_source_code(tar_file_path, source)
+        source_file_path = source
```

Comment:
Please consider renaming `source_file_path` to `data_file_path`. The current name becomes confusing when we have a compressed file there, as de facto it's not really `source` anymore at that point.